assembly/src/main/quartz/quartz.properties

```diff
@@ -297,10 +297,10 @@ org.quartz.jobStore.class = org.quartz.impl.jdbcjobstore.JobStoreTX
 #_replace_jobstore_properties
 
 org.quartz.jobStore.misfireThreshold = 60000
-org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.HSQLDBDelegate
```

Comment:
Was this just a forgotten change from before?

Reply:
Apparently, yes.  I figured it's better than misstating it though the SQL probably doesn't change for what we use it for